app/models/rbac.rb

```diff
@@ -222,7 +222,7 @@ def self.find_targets_filtered_by_parent_ids(parent_class, scope, find_options,
   end
 
   def self.find_targets_filtered_by_ids(scope, find_options, u_filtered_ids, filtered_ids)
-    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count
+    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count(:all)
```

Comment:
> If count(:all) doesn't excludes virtual attributes, I don't see how this change could hurt.

Just wanted to make a clarification about this statement:  `count(:all)` _**does**_ exclude `virtual_attributes`, and that is the desired result for the `.count` call specifically.  If they are part of the count, things blow up.

Since it is effectively changing the "`SELECT`" of the count, not what we are filtering my (the `WHERE`), it is fine to leave them out since it is just getting a baseline of the records from the DB.